language/en_us/paystack.php

```diff
@@ -4,6 +4,7 @@
 
 
 $lang['Paystack.name'] = 'Paystack Standard';
+$lang['Paystack.description'] = 'Paystack';
```

Comment:
You should probably be more descriptive with the description. Probably mention how this is used primarily in Africa